### PR TITLE
feat(presence-plugin): pass label to aphelia API whilst setting status

### PR DESCRIPTION
**Pull request by CormacGCisco**
# COMPLETES [CX-16180](https://jira-eng-sjc12.cisco.com/jira/browse/CX-16180)

## This pull request addresses

Webex user's status is not always being cleared correctly from Agent Desktop after a call ends, as a result of the `label` parameter not being filled in the request to the [Aphelia V1 API](https://backstage.corp.webex.com/catalog/default/api/apheleia-endpoint#/default/post_apheleia_api_v1_events). When the request is made without a `label`, the Aphelia backend is  unable to associate 2 events (the setting and clearing of "on a call" status) that can result in an agent's status getting "stuck" until the TTL expires

## by making the following changes

Filling the `label` parameter in the API request

### Change Type

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [ ] Documentation update
- [ ] Tooling change
- [ ] Internal code refactor

## The following scenarios were tested

* Checked request works as intended with Postman
* Added to integration test

### I certified that

- [x] I have read and followed [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request)
- [x] I discussed changes with code owners prior to submitting this pull request

- [x] I have not skipped any automated checks
- [x] All existing and new tests passed
- [ ] I have updated the documentation accordingly

---

Make sure to have followed the [contributing guidelines](https://github.com/webex/webex-js-sdk/blob/master/CONTRIBUTING.md#submitting-a-pull-request) before submitting.

### CODE REVIEW

File: packages/@webex/internal-plugin-presence/src/presence.js [hunk 1]
```diff
@@ -234,6 +234,7 @@ const Presence = WebexPlugin.extend({
         body: {
           subject: this.webex.internal.device.userId,
           eventType: status,
+		  label: this.webex.internal.device.userId,
```
Comment (Shreyas281299) [inline packages/@webex/internal-plugin-presence/src/presence.js:237]: Can we fix the indexing for all the code changes

File: packages/@webex/internal-plugin-presence/test/integration/spec/presence.js [hunk 2]
```diff
@@ -206,8 +206,10 @@ describe.skip('plugin-presence', function () {
         spock.webex.internal.presence.setStatus('dnd', 1500).then((statusResponse) => {
           assert.property(statusResponse, 'subject');
           assert.property(statusResponse, 'status');
+		  assert.property(statusResponse, 'label');
```
Comment (Shreyas281299) [inline packages/@webex/internal-plugin-presence/test/integration/spec/presence.js:209]: Can we also add unit test for the changes.

**Review by Shreyas281299 — CHANGES_REQUESTED** [review]
Please comments.

**Review by Shreyas281299 — APPROVED** [review]
Just add a line after each tests

**Review by marcin-bazyl — APPROVED** [review]

**Pull request merged**